tests/scripts/amsr2.tc_clean.89-PCT-Family.sh

```diff
@@ -54,8 +54,8 @@ run_procflow \
     $GEOIPS_TESTDATA_DIR/test_data_amsr2/data/AMSR2-MBT_v2r2_GW1_s202005180620480_e202005180759470_c202005180937100.nc \
     --procflow single_source \
     --reader_name amsr2_netcdf \
-    --product_name "89-Test" \
-    --compare_path "${GEOIPS_PACKAGES_DIR}/template_basic_plugin/tests/outputs/amsr2.tc_clean.89-Test" \
+    --product_name "89-PCT-Family" \
```

Comment:
89-PCT-Fully-Specified ?

Also, I'm guessing the output products will need to be renamed for the tests to still pass?  We will need the tests to continue passing with these updates.

Reply:
All updated